packages/app/config/env.js

```diff
@@ -18,6 +18,10 @@ const getHost = () => {
     return `http://${process.env.STAGING_BRANCH}.cs.lbogdan.tk`;
   }
 
+  if ('ROOT_URL' in process.env) {
+    return process.env.ROOT_URL;
```

Comment:
What's the difference between `ROOT_URL` and `CODESANDBOX_HOST`?

Reply:
`ROOT_URL` basically overrides the logic (and the default value of `https://codesanbox.io`) of `CODESANDBOX_HOST`.